pkgs/applications/virtualization/looking-glass-client/default.nix

```
@@ -41,9 +39,9 @@ stdenv.mkDerivation rec {
       step required to move away from dual booting with other operating systems
       for legacy programs that require high performance graphics.
     '';
-    homepage = "https://looking-glass.hostfission.com/";
+    homepage = https://looking-glass.hostfission.com/;
```

```suggestion
    homepage = "https://looking-glass.io/";
```
The website seems to be moved. Also, the unquoted URL is discouraged as per [RFC 45](https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md).

---

Done.